src/bouwdossiers/batch.py

```diff
@@ -537,6 +543,7 @@ def add_bag_ids_to_pre_wabo():
     JOIN bag_nummeraanduiding bn
         ON ba.straat = bn._openbare_ruimte_naam
         AND ba.huisnummer_van = bn.huisnummer
+        AND bn.id LIKE '0363%'
```

Comment:
Alhoewel je dit al beschreven hebt in de docstring zou ik achter deze regel ook nog een sql comment (met dubbel streepje: `-- comment hier`) zetten met danwel een korte uitleg hiervan, danwel een referentie naar de docstring.